src/dashboard/src/components/ingest/views.py

```diff
@@ -504,9 +504,16 @@ def transfer_backlog(request, ui):
     # Get search parameters from request
     results = None
 
+    # GET params in SIP arrange can control whether files in metadata/ and
+    # logs/ are returned. Appraisal tab always hides these dirs and their files
+    # (for now).
+    BACKLOG_FILTER = elasticSearchFunctions.BACKLOG_FILTER
+    if ui == 'appraisal' or request.GET.get('hidemetadatalogs'):
+        BACKLOG_FILTER = elasticSearchFunctions.BACKLOG_FILTER_NO_MD_LOGS
```

Comment:
If `BACKLOG_FILTER` is not a constant perhaps we shouldn't capitalize it. What do you think?